ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -670,12 +687,12 @@ open class BottomCommandingController: UIViewController {
         }
 
         struct BottomSheet {
-            static let expandedFraction: CGFloat = 0.7 // Probably should be more customizable / based on content
-            static let heroStackExpandableTopMargin: CGFloat = 0
-            static let heroStackNonExpandableTopMargin: CGFloat = 16
-            static let heroStackLeadingTrailingMargin: CGFloat = 8
+            static let headerHeight: CGFloat = 64
 
-            static let expandedContentTopMargin: CGFloat = 16
+            static let headerTopMargin: CGFloat = 5
```

Comment:
honestly, i don't quite grasp the 3 different top margin constants you defined her and also why headertopmargein is 5?

Reply:
Changed 5 to 4. It was mostly eyeballed from the figma spec, but 4 does feel better.